packages/devextreme/js/ui/tag_box.js

```diff
@@ -677,24 +677,13 @@ const TagBox = SelectBox.inherit({
         this.callBase(e);
     },
 
-    _getFirstPopupElement: function() {
-        return this.option('showSelectionControls')
-            ? this._list._focusTarget()
-            : this.callBase();
-    },
-
     _initSelectAllValueChangedAction: function() {
         this._selectAllValueChangeAction = this._createActionByOption('onSelectAllValueChanged');
     },
 
     _renderList: function() {
         this.callBase();
         this._setListDataSourceFilter();
-
-        if(this.option('showSelectionControls')) {
-            this._list.registerKeyHandler('tab', (e) => this._popupElementTabHandler(e));
```

Comment:
Similar handlers still exist in DropDownButton and Lookup. Shouldn't we get rid of this?

Reply:
decided to leave tab behavior as it is. esc handlers are removed